k2/addons/k2-sutevents/sutevents/suteventslog.py

```diff
@@ -107,14 +109,18 @@ def __init__(self, config, instances):
     def destroy(self):
         self.cancel_triggers()
 
+    def register_dispatchers(self, messagebus):
+        if self._serial_ports:
+            self.dispatcher = SequentialDispatcher(messagebus, self.handle_messages)
+            self.dispatcher.register([LOG_LINE_RECEIVED], entities=self._serial_ports)
+
     @callback_dispatcher([UNINITIALIZE_SUT], entity_option_id=SUT)
     def cancel_triggers(self, message=None):
         with self.pattern_handlers.lock:
             for _, trigger in self.pattern_handlers.items():
                 trigger.cancel()
 
-    @sequential_dispatcher(
-        [LOG_LINE_RECEIVED, SUT_RESET_EXPECTED, SUT_RESET_NOT_EXPECTED], entity_option_id=SUT)
+    @sequential_dispatcher([SUT_RESET_EXPECTED, SUT_RESET_NOT_EXPECTED], entity_option_id=SUT)
```

Comment:
I don't understand how this can work now when you have removed the LOG_LINE_RECEIVED. It will no longer process log-lines. Is this change really intentionally? If yes, what am I missing? How will it work?

Reply:
I now I see, you register the LOG_LINE_RECEIVED above on lines 114-115. Never mind above comment.